### ISSUE TITLE
fontawesome for icons

### ISSUE THREAD

**Issue by bbeardsley** [SSilence/selfoss#1094]
I am trying to implement a dark theme and am having trouble with icons not setup for inverted colors.  Can we switch the icons in toolbar etc over to fontawesome?  I'd be happy to work on that if that's a direction this project would like to go.

**Comment by jtojnar** [comment]
That is exactly what https://github.com/SSilence/selfoss/pull/1013 does. I want to finish it during the weekend.

My motivation is the same, here is my attempt:

```css
// :focus {
// 	border: 5px solid crimson;
// }
// .entry-smartphone-share button.entry-share,
// .entry-smartphone-share button {
//     padding: 20px;
// }
// .entry-toolbar button {
//     margin-top: 0px;
//     padding: 6px 0 6px 20px;
// }

button {
    color: white;
}

#nav-open {
    padding: 10px;
    border: none;
    border-radius: 2px;
    background-color: #272325;
    cursor: pointer;
    margin-left: 15px;
    margin-right: 15px;
    margin-top: 5px;
    font-size: 0.9em;
    text-align: center;
}

body {
    color: white;
    background-color: #2e3436;
    background-image: url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAwAAAAMAQMAAABsu86kAAAABlBMVEUUFBQMDAzbIvKLAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAB3RJTUUH4QkPDBUFCnr7XwAAACZJREFUCNdj+PmB4dsHhvcFDPc3MOy/wFD/AISADCAXKAiU+vkBAFyKE1feGt/VAAAAAElFTkSuQmCC');
}
.entry.unread .entry-title {
    color: #eeeeec;
}
.entry-title {
    color: #999999;
}
.source-edit-form {
   color: #cececc;
}
.source-title {
    color: #eeeeec;
}
.source-add {
    color: inherit;
}
.entry, .source {
    background: #2e343666;
    border: 1px solid #fff0;
}

body:not(.loggedin) #nav-filter-unread {
    display: none;
}
.entry.selected {
    background: #0D1211;
    border-color: #444F4E;
}
.entry-toolbar {
    border-top-color: #444F4E;
    background-color: #0D1211;
}

.entry-smartphone-share button, .entry-toolbar button {
    color: white;
}

.refresh-source {
    color: white;
}

#fullscreen-entry {
    background: transparent;
}
```

**Comment by jtojnar** [comment]
I still need to finish up asset caching in the Webpack powered solution, will take a look at it again on the weekend.

**Comment by jtojnar** [comment]
Fixed in #1013
